brandonrhayes/Minimizing_Vaccination_Waste
# GUI variables for TextFields and RadioButtons: Set and Get

Here's whats new:
- Variable names have been assigned for all textfields and radio buttons (set)
- The values of these variables can be accessed (get)
- These values of these variables print to the console after clicking the "See Results" button
- Variable names been set for the first dropdown, but cannot yet get these values

How to test:
- Validate that all existing functionality works as expected (dropdowns show values, "See Results" buttons takes you to results.fxml, etc.)
- Type any value into all TextFields on the primary.fxml page
- Select the "yes/no" radio button
- Click "See Results" button and validate that inputs are printed to the console as expected. See example below:
Input: 
![image](https://user-images.githubusercontent.com/55509733/105409758-e9f68480-5bfe-11eb-87bb-852173d3df12.png)

Output:
![image](https://user-images.githubusercontent.com/55509733/105409786-f67add00-5bfe-11eb-981d-f3fbc6a78720.png)
 
- Additionally, type/select nothing in textfields/radiobuttons, and validate that nothing is printed to the console:
![image](https://user-images.githubusercontent.com/55509733/105409818-04306280-5bff-11eb-8c33-e19f746731d7.png)

Thanks team!
